### DISCUSSION TITLE
不是绳匠不准点进来看！

### DISCUSSION THREAD

**Discussion by YumeRin39Hana** [share121/inter-knot#27] [其他]
![一之濑贵子](https://github.com/user-attachments/assets/6bfa45a5-b793-4c7a-955a-40679757edd8)

新到了一批驱动盘，有意者+13653811私聊，军方那边的货，质量保障就对到位！
.........以上是做的推销.............
现在当绳匠真是太难了，上有治安局追捕，下有刁蛮委托人，时不时还被某个放羊的给坑两下，靠！我是来做慈善的吗？
有的委托人接的时候好好的，委托刚结束就开始坐地起价，开口第一句就是“你又不是传奇绳匠法厄同，干嘛收费这么贵？”大哥大姐，松松口饶了我吧，上班累死累活，拿那么一点喂狗都嫌的薪水，听一人全家福的老板给我望梅止渴，有空闲了上绳网做做委托干点副业还被这么刁难，我要是能做成法厄同，简单的委托我免费帮你做行不行啊？

**Comment by Zzz-walnut** [comment]
驱动盘价位合适的话我要了

**Comment by YumeRin39Hana** [comment]
B级1500丁尼/1个，A级3500丁尼/1个，S级7000丁尼/1个

**Comment by MouxiuSama** [comment]
现在绳匠不好当啊，法厄同最近频繁出现在空洞，已经引起了治安局的注意了，对了，小心三号空洞那边，而且据说绳网里埋伏了治安局里的人，现在都自身难保咯~

**Comment by ggdwj233** [comment]
怎么这里也有打广告的，各位绳匠不要轻易相信，最近很多打着军方名义的骗子，说是上乘的驱动盘，结果到手全是粗制滥造的玩意，我一个好友已经被这种套路骗了好几次了

**Comment by koyki233** [comment]
不是哥们，你咋不去抢呢，我还得攒着100亿丁尼买房呢，买不起买不起
